inferlo/pairwise/optimization/convex_bounds_test.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
```

Comment:
General advice: good tests should fail if code is incorrect. We want to test that approximation is good enough and it works on all kinds of graphs.

I suggest:
1. Have at least a simple test for small model build "by hand" (without rndom generator), for which relaxation has single global optimum and explictly assert (using np.allclose) that optimum that it returns is equal to what we expect. Also assert that objective value is close.
2. Have a test for which we easily know what is the most likely state. For example, model where fields are much stronger than interactions, then max likelihood state is just argmax over fields. Then assert that "rounded" solution gives expected result. It not always is true, but for simple enough case this should work.
3. You can also leave tests you already have that check that solution of relaxation is >= than ground truth, but for that you need to pass objective in namedtuple (see another comment).